phpunit/phpunit/4.7/post-install.txt

```diff
@@ -1,7 +1,7 @@
-<bg=yellow;fg=black>                                                        </>
-<bg=yellow;fg=black> Adding phpunit/phpunit as a dependency is discouraged. </>
-<bg=yellow;fg=black>                                                        </>
+<bg=yellow;fg=black>                                                                                              </>
```

Comment:
looks like you now need to fix the alignment (i.e. removing one space here and two lines below)